graphql/schema/wrappers.go

```diff
@@ -343,6 +344,13 @@ func (o *operation) Mutations() (ms []Mutation) {
 	return
 }
 
+func (o *operation) CacheControl() string {
+	if o.op.Directives.ForName(cacheControlDirective) == nil {
+		return ""
```

Comment:
private,no-cache would work instead of empty string

Reply:
adding header only if the directive is there, otherwise no header.